src/test/java/ru/ayaz/VideoTest.java

```diff
@@ -0,0 +1,39 @@
+package ru.ayaz;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+
+public class VideoTest {
+
+    private Video video;
+
+    @Before
+    public final void before() {
+        this.video = new Video("FunnyCats", mock(File.class));
+    }
+
+    @Test
+    public void shouldContainCommentInListWhenAddedToVideo() {
+        Comment comment = mock(Comment.class);
+
+        video.addComment(comment);
+
+        assertTrue(video.getComments().contains(comment));
```

Comment:
&#x1F34E; **Issue**
Here you ask for the object first (.getComments()) and then perform action on that object (.contains(comment)). The preferred way might be to ask "video" to check whether or not it has a comment instead of doing two sequential calls yourself. (Remember about the door and a handle example. You don't get the handle and open it in order to open a door. You just open the door.)

And as I see below, you have methods like "deleteComment" and "addComment". Why didn't you there just get the  list of comments and acted on it ?)